int16 rearSignal
int16 leftSignal
int16 rightSignal
